src/ResourceManager/RecoveryServices/Commands.RecoveryServices.SiteRecovery/RecoveryPlan/EditAzureRmRecoveryServicesAsrRecoveryPlan.cs

```diff
@@ -24,7 +24,11 @@ namespace Microsoft.Azure.Commands.RecoveryServices.SiteRecovery
     /// <summary>
     ///     Edits a ASR Recovery plan in memory.
     /// </summary>
-    [Cmdlet("Edit", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "RecoveryServicesAsrRecoveryPlan",DefaultParameterSetName = ASRParameterSets.AppendGroup,SupportsShouldProcess = true)]
+    [Cmdlet(
+        VerbsData.Edit,
+        "AzureRmRecoveryServicesAsrRecoveryPlan",
```

Comment:
Needs to be `ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "RecoveryServicesAsrRecoveryPlan"`